pytentiostat/config_reader.py

```diff
@@ -42,6 +42,25 @@ def parse_config_file(configlocation=None):
         sys.exit("Directory containing config file, {}, not found. Exiting...".format(configlocation))
 
 def get_output_params(config_data, override_ts=None):
+    """
+    Returns the output filename and output filepath from the config file data.
+
+    Parameters
+    ----------
+    config_data : dict
+        The configuration data
+    override_ts : (str, optional)
+        Unique string to append to output filename string.  If not specified will add a time stamp
+        hours, minutes, and seconds the experiment was conducted at as a unique identifier.
+
+    Returns
+    -------
+    out_name_ts : str
+        The output filename with unique identifier.    
```

Comment:
a little unclear, this.   maybe a second sentence that discusses how the name is composed rather than just "with unique identifier"?

Reply:
Added what unique identifier is added to the file